src/main/webapp/timeline.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Timeline</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+     <img src="images/colorBlockStripe.jpg" class="stripe"/>
+     <div>
+         <img src="images/profpic.jpg" class="profPicDefault"/>
+         <h1>Name</h1>
+         <h2>Skate Type</h2>
+         <p class=bioText>Bio sample: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+         <a href="editBio.html" class="editButton">Edit</a>
```

Comment:
use lower case for html file names. If you want to keep the words separate, consider using dashes.

Reply:
Got it!! Thanks